src/native/common.gypi

```diff
@@ -18,10 +18,13 @@
                     '-fno-exceptions',
                 ],
                 'cflags': [
+                    '-Wno-cast-function-type',
```

Comment:
How many of these do we have? Maybe it would be best not to ignore and add explicit cast if needed.

Reply:
Removing this flag we have:
12 `-Wimplicit-fallthrough=` all in `gf256.cpp`

2 `-Wsign-compare in `snappy.cc`

1 `-Wsign-compare in `utp_internal.cpp`

1 `-Werror=cast-function-type` in `node.h`

**another note:** is when we have the `'-Wno-cast-function-type',` flag we don't need the  `'-Werror'` in the `disabled_warnings` on  `common_third_party.gypi`